app/forms/provider_interface/offer_wizard.rb

```diff
@@ -8,13 +8,15 @@ class OfferWizard
     attr_accessor :provider_id, :course_id, :course_option_id, :study_mode, :location_id,
                   :standard_conditions, :further_condition_1, :further_condition_2,
                   :further_condition_3, :further_condition_4, :current_step, :decision,
-                  :path_history, :action, :provider_user_id, :wizard_path_history
+                  :action, :path_history, :wizard_path_history,
+                  :provider_user_id, :application_choice_id
 
     validates :decision, presence: true, on: %i[select_option]
     validates :course_option_id, presence: true, on: %i[locations save]
     validates :study_mode, presence: true, on: %i[study_modes save]
     validates :course_id, presence: true, on: %i[courses save]
     validates :further_condition_1, :further_condition_2, :further_condition_3, :further_condition_4, length: { maximum: 255 }
+    validates :application_choice_id, presence: true
```

Comment:
We don't want to render an application choice error to the provider_user (as it's not something they can address), so we should not check for presence here. `find` on line 98 will cause a resource not found error which is what we want.